kdc/kerberos5.c

```diff
@@ -60,6 +60,123 @@ realloc_method_data(METHOD_DATA *md)
     return 0;
 }
 
+extern int _krb5_AES_SHA1_string_to_default_iterator;
+extern int _krb5_AES_SHA2_string_to_default_iterator;
+
+static krb5_error_code
+make_s2kparams(int value, size_t len, krb5_data **ps2kparams)
```

Comment:
A forward declaration would have resulted in less code churn, but I'm OK with this.

Reply:
Actually, let's go with a forward decl to avoid churn.

Reply:
Please, yes.